app/facades/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+  def self.total_results(keyword)
+    @total_results = FoodService.total_results(keyword)
+  end
+
+  def self.top_10_results(keyword)
+    @top_10_results = FoodService.top_10_results(keyword)
+  end
+end
```

Comment:
You can pull the total results number out of the call you're performing on line 7. Do that instead of making 2 api calls.